sync/src/main/java/tech/pegasys/teku/sync/forward/multipeer/MultipeerSyncService.java

```diff
@@ -91,7 +93,7 @@ public static MultipeerSyncService create(
                 pendingBlocks,
                 finalizedTargetChains,
                 nonfinalizedTargetChains,
-                Constants.SLOTS_PER_EPOCH),
+                spec.getSlotsPerEpoch(recentChainData.getFinalizedEpoch())),
```

Comment:
getSlotsPerEpoch takes in a slot. Probably worth calling `spec.getSlotsPerEpoch(recentChainData.getCurrentSlot())`